test/Microsoft.AspNetCore.Mvc.IntegrationTests/ArrayModelBinderIntegrationTest.cs

```diff
@@ -327,5 +327,41 @@ private class Person
             Assert.Equal(0, modelState.ErrorCount);
             Assert.True(modelState.IsValid);
         }
+
+        private class PersonWithReadOnly
+        {
+            public string Name { get; set; }
+
+            public string[] Aliases { get; }
```

Comment:
I have modified existing test to have non-null values instead since that is more of a possible scenario than a null read only property...i have done similar updates to other test